rs/nervous_system/agent/src/state_machine_impl.rs

```diff
@@ -130,13 +125,4 @@ impl CallCanisters for StateMachine {
             .canister_info(canister_id)
             .await
     }
-
-    fn is_canister_stopped_error(&self, err: &Self::Error) -> bool {
-        match err {
-            StateMachineCallError::IngressStateError(err) => {
-                [ErrorCode::CanisterStopped, ErrorCode::CanisterStopping].contains(&err.code())
-            }
-            _ => false,
-        }
-    }
```

Comment:
Where is this logic moved to?

Reply:
It was removed altogether since it wasn't used anywhere.
`sns-testing` uses only `Agent` and `PocketIc` implementations for this trait

Reply:
According to IC devs, `StateMachine` is used for legacy tests, so it's fine to omit the implementation for `CallCanistersWithStoppedCanisterError`